packages/fast-data-utilities-react/src/index.ts

```diff
@@ -278,6 +325,95 @@ function getLocationsFromObject(data: any, location: string = ""): string[] {
     return updatedLocations;
 }
 
+/**
+ * Callback to determine if a string is found within an array of plugin locations
+ */
+function pluginPartialFindIndexCallback(location: string): any {
```

Comment:
I think you have enough info to accuratly type the return value of this function